policybot/pkg/storage/types.go

```diff
@@ -75,13 +77,15 @@ type Org struct {
 	Company     string
 	AvatarURL   string
 	Description string
+	Config      config.Org `spanner:"-"`
 }
 
 type Repo struct {
 	OrgLogin    string
 	RepoName    string
 	Description string
 	RepoNumber  int64
+	Config      config.Repo `spanner:"-"`
```

Comment:
I really would rather not start mixing layers here by storing config information in the storage layer's data structures. Can this be handled a different way?